projects.py

```diff
@@ -30,10 +32,21 @@ def parse_args():
 
 
 def run_projects(c, project_name=None):
+    projects_start_time = datetime.datetime.now()
+    timestamp = projects_start_time.strftime("%y%m%d")
```

Comment:
Why use a different timestamp format in linkage-agent vs data-owner-tools?

https://github.com/mitre/data-owner-tools/blob/017bff99c3c62e61849de9bbfd16dcfa9b8ea39c/extract.py#L263

`"%Y%m%dT%H%M%S"`